### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by DTMad**

### CODE REVIEW

File: pkg/integrationtests/envutils_test.go [hunk 1]
```diff
@@ -116,12 +116,18 @@ func newTestEnvironment() (*ControllerTestEnvironment, error) {
 
 	kubernetesClient, err := client.New(cfg, client.Options{Scheme: scheme.Scheme})
 	if err != nil {
-		kubernetesAPIServer.Stop()
+		err = kubernetesAPIServer.Stop()
+		if err != nil {
+			return nil, err
```
Comment (lrgar) [inline pkg/integrationtests/envutils_test.go:121]: You're ignore the error from `kubernetesClient.Create()`. You can add the two errors into the same error message.

File: pkg/integrationtests/envutils_test.go [hunk 2]
```diff
@@ -116,12 +116,18 @@ func newTestEnvironment() (*ControllerTestEnvironment, error) {
...
+		}
 		return nil, err
 	}
 
 	if err = kubernetesClient.Create(context.TODO(), buildDynatraceClientSecret()); err != nil {
-		kubernetesAPIServer.Stop()
+		err = kubernetesAPIServer.Stop()
+		if err != nil {
+			return nil, err
```
Comment (lrgar) [inline pkg/integrationtests/envutils_test.go:129]: Same here.

File: pkg/dtclient/dynatrace_client.go [hunk 3]
```diff
@@ -109,7 +109,7 @@ func (dc *dynatraceClient) getHostInfoForIP(ip string) (*hostInfo, error) {
 }
 
 func (dc *dynatraceClient) buildHostCache() error {
-	var url string = fmt.Sprintf("%s/v1/entity/infrastructure/hosts?includeDetails=false", dc.url)
+	var url = fmt.Sprintf("%s/v1/entity/infrastructure/hosts?includeDetails=false", dc.url)
```
Comment (meik99) [inline pkg/dtclient/dynatrace_client.go:112]:
```suggestion
	url := fmt.Sprintf("%s/v1/entity/infrastructure/hosts?includeDetails=false", dc.url)
```

File: pkg/dtclient/dynatrace_client_test.go [hunk 4]
```diff
@@ -31,7 +31,7 @@ func TestMakeRequest(t *testing.T) {
 	require.NotNil(t, dc)
 
 	{
-		var url string = fmt.Sprintf("%s/v1/deployment/installer/agent/connectioninfo", dc.url)
+		var url = fmt.Sprintf("%s/v1/deployment/installer/agent/connectioninfo", dc.url)
```
Comment (meik99) [inline pkg/dtclient/dynatrace_client_test.go:34]:
```suggestion
		url := fmt.Sprintf("%s/v1/deployment/installer/agent/connectioninfo", dc.url)
```

File: pkg/webhook/bootstrapper/controller.go [hunk 5]
```diff
@@ -230,9 +226,9 @@ func (r *ReconcileWebhook) reconcileWebhookConfig(ctx context.Context, log logr.
 
 	scope := admissionregistrationv1beta1.NamespacedScope
 	path := "/inject"
-	webhook := &admissionregistrationv1beta1.MutatingWebhookConfiguration{
+	wh := &admissionregistrationv1beta1.MutatingWebhookConfiguration{
```
Comment (meik99) [inline pkg/webhook/bootstrapper/controller.go:229]:
```suggestion
	webhookConfiguration := &admissionregistrationv1beta1.MutatingWebhookConfiguration{
```

File: pkg/webhook/bootstrapper/controller.go [hunk 6]
```diff
@@ -268,11 +264,11 @@ func (r *ReconcileWebhook) reconcileWebhookConfig(ctx context.Context, log logr.
 	}
 
 	var cfg admissionregistrationv1beta1.MutatingWebhookConfiguration
-	err := r.client.Get(context.TODO(), client.ObjectKey{Name: webhookName}, &cfg)
+	err := r.client.Get(context.TODO(), client.ObjectKey{Name: webhook.ServiceName}, &cfg)
 	if k8serrors.IsNotFound(err) {
 		log.Info("MutatingWebhookConfiguration doesn't exist, creating...")
 
-		if err = r.client.Create(ctx, webhook); err != nil {
+		if err = r.client.Create(ctx, wh); err != nil {
```
Comment (meik99) [inline pkg/webhook/bootstrapper/controller.go:271]:
```suggestion
		if err = r.client.Create(ctx, webhookConfiguration); err != nil {
```

File: pkg/webhook/bootstrapper/controller.go [hunk 7]
```diff
@@ -287,6 +283,6 @@ func (r *ReconcileWebhook) reconcileWebhookConfig(ctx context.Context, log logr.
 	}
 
 	log.Info("MutatingWebhookConfiguration is outdated, updating...")
-	cfg.Webhooks = webhook.Webhooks
+	cfg.Webhooks = wh.Webhooks
```
Comment (meik99) [inline pkg/webhook/bootstrapper/controller.go:286]:
```suggestion
	cfg.Webhooks = webhookConfiguration.Webhooks
```

**Review by meik99 — APPROVED** [review]
Some suggestions but nothing critical

File: pkg/integrationtests/envutils_test.go (hunk 1 above)
Comment (DTMad) [reply pkg/integrationtests/envutils_test.go]: Done

File: pkg/integrationtests/envutils_test.go (hunk 2 above)
Comment (DTMad) [reply pkg/integrationtests/envutils_test.go]: Done

File: CHANGELOG.md [hunk 8]
```diff
@@ -18,6 +18,7 @@
 
 #### Other changes
 * Pod and node metadata added for the OneAgent ([#294](https://github.com/Dynatrace/dynatrace-oneagent-operator/pull/294), [#295](https://github.com/Dynatrace/dynatrace-oneagent-operator/pull/295))
+* Code cleanup to remove unused functions, variables and beautify the code
```
Comment (lrgar) [inline CHANGELOG.md:21]:
```suggestion
* Code cleanup to remove unused functions, variables and beautify the code ([#302](https://github.com/Dynatrace/dynatrace-oneagent-operator/pull/302))
```

**Review by lrgar — APPROVED** [review]

**Pull request merged**